{
  "name": "CVE-2019-14433",
  "seq": "2019-14433",
  "type": "CAN",
  "status": "Candidate",
  "phase": {
    "text": "Assigned",
    "date": "20190729"
  },
  "desc": "An issue was discovered in OpenStack Nova before 17.0.12, 18.x before 18.2.2, and 19.x before 19.0.2. If an API request from an authenticated user ends in a fault condition due to an external exception, details of the underlying environment may be leaked in the response, and could include sensitive configuration or other data.",
  "refs": [
    {
      "text": "https://security.openstack.org/ossa/OSSA-2019-003.html",
      "source": "CONFIRM",
      "url": "https://security.openstack.org/ossa/OSSA-2019-003.html"
    },
    {
      "text": "https://launchpad.net/bugs/1837877",
      "source": "MISC",
      "url": "https://launchpad.net/bugs/1837877"
    },
    {
      "text": "[debian-lts-announce] 20220915 [SECURITY] [DLA 3109-1] nova security update",
      "source": "MLIST",
      "url": "https://lists.debian.org/debian-lts-announce/2022/09/msg00018.html"
    },
    {
      "text": "[oss-security] 20190806 [OSSA-2019-003] Nova Server Resource Faults Leak External Exception Details (CVE-2019-14433)",
      "source": "MLIST",
      "url": "http://www.openwall.com/lists/oss-security/2019/08/06/6"
    },
    {
      "text": "RHSA-2019:2622",
      "source": "REDHAT",
      "url": "https://access.redhat.com/errata/RHSA-2019:2622"
    },
    {
      "text": "RHSA-2019:2631",
      "source": "REDHAT",
      "url": "https://access.redhat.com/errata/RHSA-2019:2631"
    },
    {
      "text": "RHSA-2019:2652",
      "source": "REDHAT",
      "url": "https://access.redhat.com/errata/RHSA-2019:2652"
    },
    {
      "text": "USN-4104-1",
      "source": "UBUNTU",
      "url": "https://usn.ubuntu.com/4104-1/"
    }
  ],
  "votes": {}
}
